arelle/Validate.py

```diff
@@ -734,7 +734,7 @@ def determineTestStatus(self, modelTestcaseVariation, errors, validateModelCount
             if not isinstance(expected, list):
                 expected = [expected]
             for testErr in _errors:
-                if isinstance(testErr,str) and testErr.startswith("ESEF."): # compared as list of strings to QName localname
+                if isinstance(testErr,str) and (testErr.startswith("ESEF.") or testErr.startswith("NL.NL-KVK")): # compared as list of strings to QName localname
```

Comment:
`startswith()` can take a tuple of prefixes:
```suggestion
                if isinstance(testErr,str) and testErr.startswith("ESEF.", "NL.NL-KVK"): # compared as list of strings to QName localname
```

Reply:
Oh sweet! TIL